### DISCUSSION TITLE
Brewed apps doesn't auto start after user login

### DISCUSSION THREAD

**Discussion by dbknv** [Homebrew/discussions#102] [Everyday usage]
### What you were trying to do (and why)
I'm trying to install application via brew. After that I'd like to enable this application to autostart. For example gSwitch (but I have the same behaviour with BetterTouchTool, CleanShotX, Bartender and so on)

System Preferences -> Users -> Login Items
or
Inside gSwitch application
**Non of those isn't working. gSwitch doesn't start after user login.**

### What you expected to happen
After I installed brewed app to /Application it will be possible to enable autostart this app via in-app setting or User login item in system preferences.

Brew installs application without any errors
```
brew cask install gSwitch

==> Downloading https://github.com/CodySchrank/gSwitch/releases/download/1.9.7/gSwitch.zip
Already downloaded: /Users/wa-bakanov/Library/Caches/Homebrew/downloads/08f286c4e002486ae3fef782d78d11091949c32195389fe7193a3361b40c1567--gSwitch.zip
==> Verifying SHA-256 checksum for Cask 'gswitch'.
==> Installing Cask gswitch
==> Moving App 'gSwitch.app' to '/Applications/gSwitch.app'.
🍺  gswitch was successfully installed!
```

I have last update 
```
brew doctor
Your system is ready to brew.


brew config
HOMEBREW_VERSION: 2.5.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: abf214c01ff1edb57acf7e23d5eb24a777335e11
Last commit: 9 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 0a483dc47a7f30266b626f1bfcf465c733c880b8
Core tap last commit: 18 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 12.0 build 1200
Git: 2.29.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 10.15.7-x86_64
CLT: 12.0.0.0.1.1599194153
Xcode: 12.0.1
```

**Comment by SMillerDev** [comment]
Brew cask only runs the installer that the creator provides. It doesn't do anything with startup scripts etc.
